tests/rms-harness/tests/test_harness

```diff
@@ -12,7 +12,7 @@ check_rms
 rm=$RESOURCE_MANAGER
 
 NODES=2
-TASKS=2
+TASKS=`tasks_count 2`
```

Comment:
Which .spec should be added to spec_to_test_mapping.py for this test ?

Reply:
maybe always?

Reply:
Yes, something like this. Not sure myself.

Reply:
Done!
We can remove it if it causes any problems!

Reply:
It failed on RHEL.
It passes fine here (on openEuler).
I'm trying to find the reason.

Reply:
The error is:
```
-------------------------------------------------------
User Environment: RMS Harness tests: gnu12-openmpi4
-------------------------------------------------------
Lmod has detected the following error: The following module(s) are unknown:
"prun"
```